### PR TITLE
Typography

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
Please review my component!

### CODE REVIEW

File: Components/Typography/typography.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  padding: 1.5rem;
+  border-radius: 6px;
+  margin-bottom: 1.5rem;
+}
+.type-1,
+.type-2,
+.type-3,
+.type-4,
+.type-5,
+.type-6 {
```
Comment (maitrakhatri) [inline Components/Typography/typography.css]: better naming can be done, `type` doesn't tell it's for font or image or what. If font then what type of font heading, regular, small

**Review by maitrakhatri — COMMENTED** [review]
just the one thing mentioned above, rest looks good

File: Components/Typography/typography.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  margin-bottom: 1.5rem;
+}
+.heading-1,
+.heading-2,
+.heading-3,
+.heading-4,
+.heading-5,
+.heading-6 {
+  margin: 1rem;
+}
```
Comment (rohanmathur91) [inline Components/Typography/typography.css:15]: you can have a utility class for margin, just a suggestion you can ignore if you want.

**Review by rohanmathur91 — APPROVED** [review]
Looks good to me.

**Pull request merged**